README.md

```diff
@@ -166,14 +166,30 @@ The `GET /metrics` endpoint will be exposed by default on port `8080`, which you
 
 ### Local setup
 
+#### Prometheus
+
 1. Start the relayer with metrics enabled
 2. Spin up the Prometheus instance:
    ```sh
    docker run -it -v $(pwd):/prometheus -p9090:9090 prom/prometheus --config.file=demo/prometheus.yaml
    ```
-   > **NOTE:** Ensure that `the --config.file=<path>` flag points at the existing configuration file. You can find an example here: [prometheus.yaml](demo/prometheus.yaml).
+   > **NOTE:** Ensure that `the --config.file=<path>` flag points at the existing configuration file. If you wish to use [the example config](demo/prometheus.yaml), just run the command above in the root of this repository. Otherwise, you must adjust the volume (`-v`) and config file path to your setup.
 3. Open the Prometheus dashboard in a browser at [http://localhost:9090](http://localhost:9090)
 
+#### Grafana
+
+1. Spin up the Grafana instance:
+   ```sh
+   docker run -d --name=grafana -p 3000:3000 grafana/grafana
+   ```
+2. Navigate to [http://localhost:3000](http://localhost:3000) and log in (`admin`/`admin`)
+3. [Create a Prometheus data source](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-data-source)
+   > **NOTE:** Use `http://localhost:9090/` as the server URL, and `Browser` as Access method for the local setup.
```

Comment:
Are you sure this works?
If I want to connect one docker container from another, I usually need something like `--networking=host` or use a different name than localhost (one pointing to the host, not the docker guest)

Reply:
`http://localhost:9090` with the `Browser` access method works as the client browser is doing the ping pong between Prometheus and Grafana. That is okish for local installations but must not be used in production for security reasons. 
You can also use `http://host.docker.internal:9090` with access type `Server` to let Grafana do the queries

Please add a note that this setup is for local testing only, to be on the safe side

Reply:
Nice to know. I didn't realize there was browser ping-pong.

Reply:
I like the server comment, as that makes more sense for my mental model, but the other one is cool too

Reply:
I also find this (Server method) more concise. There is no need to mention the `Browser` access method, so I changed the note to use the `Server` instead: 1158237

Thanks for bringing this!